### PR TITLE
Clean up world Reboot, reorganize sounds

### PR DESCRIPTION
**Pull request by Cyberboss**
New folder for round end sounds

Deleted newroundsexy2.ogg, unused, not sexy.

### CODE REVIEW

File: code/world.dm [hunk 1]
```diff
@@ -234,36 +233,40 @@ var/last_irc_status = 0
...
 	for(var/thing in clients)
 		var/client/C = thing
-		if (!C)
-			continue
-		if(config.server)	//if you set a server location in config.txt, it sends you there instead of trying to reconnect to the same world address. -- NeoFite
+		if(C && config.server)	//if you set a server location in config.txt, it sends you there instead of trying to reconnect to the same world address. -- NeoFite
 			C << link("byond://[config.server]")
 
+/world/proc/RoundEndSound(round_end_sound_sent)
+	set waitfor = FALSE
```
Comment (Cyberboss) [inline code/world.dm]: Does `sound()` sleep? If not, I don't know why it was in a spawn before

**Comment by AnturK** [comment]
Said this before but roundend sounds should be server config.

File: code/world.dm [hunk 2]
```diff
@@ -225,7 +225,6 @@ var/last_irc_status = 0
 /world/proc/OnReboot(reason, feedback_c, feedback_r, round_end_sound_sent)
 	feedback_set_details("[feedback_c]","[feedback_r]")
 	log_game("<span class='boldannounce'>Rebooting World. [reason]</span>")
-	kick_clients_in_lobby("<span class='boldannounce'>The round came to an end with you in the lobby.</span>", 1) //second parameter ensures only afk clients are kicked
```
Comment (Iamgoofball) [inline code/world.dm]: Nice, thanks

**Review by Iamgoofball — APPROVED** [review]

File: code/world.dm [hunk 3]
```diff
@@ -234,36 +233,40 @@ var/last_irc_status = 0
...
-			'sound/misc/leavingtg.ogg',
-			'sound/misc/its_only_game.ogg',
-			'sound/misc/yeehaw.ogg',
-			'sound/misc/disappointed.ogg')) // random end sounds!! - LastyBatsy
 	if(blackbox)
 		blackbox.save_all_data_to_sql()
-	sleep(soundwait)
 	Master.Shutdown()	//run SS shutdowns
+	RoundEndSound(round_end_sound_sent)
+	kick_clients_in_lobby("<span class='boldannounce'>The round came to an end with you in the lobby.</span>", 1) //second parameter ensures only afk clients are kicked
```
Comment (Cyberboss) [inline code/world.dm]: Nice try @Iamgoofball

Comment (Iamgoofball) [inline code/world.dm]: Fuck

**Comment by Cyberboss** [comment]
@AnturK Configurable rsc files though?

**Comment by Iamgoofball** [comment]
We can just send them to players

File: code/world.dm (hunk 2 above)
Comment (PKPenguin321) [inline code/world.dm]: mekhi_porn.txt

File: code/world.dm [hunk 4]
```diff
@@ -234,36 +233,40 @@ var/last_irc_status = 0
...
-		if (!C)
-			continue
-		if(config.server)	//if you set a server location in config.txt, it sends you there instead of trying to reconnect to the same world address. -- NeoFite
+		if(C && config.server)	//if you set a server location in config.txt, it sends you there instead of trying to reconnect to the same world address. -- NeoFite
 			C << link("byond://[config.server]")
 
+/world/proc/RoundEndSound(round_end_sound_sent)
+	set waitfor = FALSE
+	var/round_end_sound
+	if(!ticker && ticker.round_end_sound)
```
Comment (kevinz000) [inline code/world.dm]: If .... no ticker... and the nonexistant ticker... has a round end sound?

**Pull request merged**